packages/xo-server/src/api/vif.js

```diff
@@ -97,6 +97,11 @@ export async function set({
       }
     }
 
+    // - If locking mode has explicitly passed: use it
+    // - Else if the network is changing: config it to 'network_default'
+    // - Else: use the old locking mode
+    lockingMode = lockingMode ?? (isNetworkChanged ? 'network_default' : vif.lockingMode)
```

Comment:
No need to reassign if already defined, either inline this or use an `if` statement :slightly_smiling_face: